odoo/tools/translate.py

```diff
@@ -367,7 +408,12 @@ def get_text_content(term):
     content = html.fromstring(term).text_content()
     return " ".join(content.split())
 
+def is_text(term):
+    """ Return whether the term has only text. """
+    return len(html.fromstring(f"<_>{term}</_>")) == 0
```

Comment:
Is ther any benefit for using `<_>` instead of `<div>`. If no I prefer using `<div>` like other codes
and you should bind it to `xml_translate` and `html_translate`

Reply:
I didn't want to unintentionally close a non-closed html tag. `<div>xxx` is valid HTML:
```
>>> len(html.fromstring("<div>xxx"))
0
>>> len(html.fromstring("<div><div>xxx</div>"))
1
```
You may argue that that's overkill, you may be right, but it doesn't hurt to use `_` as a low-changes-of-closing-anything tag :) You can let me know if you still prefer `div` :)

In any case, I'll bind it to the translate functions :)

Reply:
This other case is NOT the same:
```
>>> term="</div>xxx"
>>> len(html.fromstring(f"<_>{term}</_>"))
0
>>> len(html.fromstring(f"<div>{term}</div>"))
1
```

Reply:
OK. Since `<_>` will not add extra overhead, and in case someone may reuse the function in other places, we can keep it.